app/script_variant.py

```diff
@@ -65,6 +69,51 @@ def romanize_japanese_name_by_name_dict(word):
     return [word]
 
 
+def romanize_japanese_name_by_name_dict(word, for_index=True):
+    """
+    This method romanizes japanese name by using name dictionary.
+    If word isn't found in dictionary, this method doesn't
+    apply romanize.
+    This method can return multiple romanizations.
+    (because there are multiple ways to read the same kanji name in japanese)
+    Args:
+        for_index: this method is called for indexing or not
+                   when you want to romanize query txt, set this False
+    Returns:
+        [romanized_jp_name, and romanized_jp_name(split_word), ...]
+    """
+    if not word:
+        return ['']
+
+    words = []
+    for index in xrange(1, len(word)):
+        # split word because query word may not contains white space
+        # to support romanizing fullname without space.
+        # e.g., if the query (e.g., "山田太郎") doesn't contain white space,
```

Comment:
It should be something like:

> If the query is a full name without white space (e.g., "山田太郎"), and the first/last name is in the dictionary, but the full name is not in the dictionary, it can still return romanization "yamadataro".